pycloudlib/lxd/cloud.py

```diff
@@ -21,6 +20,10 @@ class _BaseLXD(BaseCloud):
     _releases_remote = "ubuntu"
     _lxd_instance_cls = LXDInstance
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._is_container = True
```

Comment:
Does it make more sense to make this a class variable with just an annotation but no initialization? Then the child classes can override the value without needing a default here.

Reply:
I think it makes more sense to declare `_BaseLXD` as an abstract class and type-annotate `_is_container` within it. I have done that, let me know if you disagree.